### ISSUE TITLE
VGG16 on Cifar10

### ISSUE THREAD

**Issue by ZhuweiQin** [CSAILVision/NetDissect#13]
This framework works perfectly!  I was wondering if this framework can possibly examine the VGG16 trained on the cifar10 dataset?

**Comment by gauenk** [comment]
Yes, this github repo can run "network dissection" on any deep learning model if the files are provided properly to the code. You will first have to get VGG16 trained on cifar10 to run the processes.

**Comment by rahimentezari** [comment]
@ZhuweiQin Could you use it for VGG on CIFAR10?
